docs/api.md

```diff
@@ -16,23 +16,19 @@ for invoking function. By default Events API runs on `:4000` port.
 
 ### Event Definition
 
-All data that passes through the Event Gateway is formatted as an Event, based on our default Event schema:
-
-* `event` - `string` - the event name
-* `id` - `string` - the event's instance universally unique ID (provided by the event gateway)
-* `receivedAt` - `number` - the time (milliseconds) when the Event was received by the Event Gateway (provided by the event gateway)
-* `data` - type depends on `dataType` - the event payload
-* `dataType` - `string` - the mime type of `data` payload
+All data that passes through the Event Gateway is formatted as a CloudEvent, based on [CloudEvents v0.1 schema](https://github.com/cloudevents/spec/blob/master/spec.md):
```

Comment:
@alexdebrie can you talk a final look here. Should we add something?

Reply:
@mthenw @elsteelbrain Docs are fine by me